tools/PI/DevHome.PI/Helpers/WindowHelper.cs

```diff
@@ -632,4 +628,83 @@ internal static async void ShowTimedMessageDialog(FrameworkElement frameworkElem
             _log.Error(ex, "Error showing timed message dialog");
         }
     }
+
+    internal enum CpuArchitecture

```

Comment:
>CpuArchitecture [](http://example.com/codeflow?start=18&length=15)

Looks like we only use this data type for a string representation? Should GetTargetArchitecture just return a string?

Reply:
We're using a string for now, but I can easily imagine we'd want the enum for other purposes later.

Reply:
Any advantage to defining our own enum vs just using IMAGE_FILE_MACHINE?

Reply:
Yes, because our enum's string representation can be used directly. Eg, "X64" vs "IMAGE_FILE_MACHINE_AMD64".